src/tests/multihost/alltests/test_kcm.py

```diff
@@ -52,3 +52,36 @@ def test_client_timeout(self, multihost, backupsssdconf):
                                                    " /var/log/sssd/"
                                                    "sssd_kcm.log")
         assert 'Terminated client' in grep_cmd.stdout_text
+
```

Comment:
We can add marker here above the function
@pytest.mark.no_tier

And as this is a new marker, please update the alltests/pytest.ini file